tardis_env3.yml

```diff
@@ -38,7 +37,7 @@ dependencies:
   - matplotlib-base
   - ipywidgets
   - plotly
-  - qgrid
+  - qgridnext
```

Comment:
This won't work because qgridnext is not installable through conda. I got an error when updating/creating tardis env. Explicitly define to use pip:

```suggestion
  - pip:
    - qgridnext
```